src/host/mod.rs

```diff
@@ -8,6 +8,8 @@ use crate::register;
 use crate::MacAddress;
 use embedded_nal::Ipv4Addr;
 
+pub static UNSPECIFIED_IP: Ipv4Addr = Ipv4Addr::new(0, 0, 0, 0);
```

Comment:
Newer version of the `no-std-net` removed the old associated function and replaced it with const iirc, however, this ensures support for both 0.5 and 0.6 no-std-net when embedded_nal gets updated with 0.6